scripts/lint_site.sh

```diff
@@ -18,10 +18,22 @@ set -e
 
 FAILED=0
 
+BRANCH="${BRANCH:-master}"
+
 if [[ "$#" -ne 0 ]]; then
     LANGS="$*"
 else
```

Comment:
```suggestion
elif [[ ! -z "${BRANCH} ]]; then
    LANGS="en zh"
else
```

Reply:
Actually, on second thought, maybe the -z BRANCH thing should be the first check. If it's set, use it, otherwise same behavior as before:
```
if [[ ! -z "${BRANCH} ]]; then
    files_changed="$(git diff --name-only --diff-filter=b "$(git merge-base HEAD "$BRANCH")")"
    if (echo "${files_changed}" | grep -q /en/); then
        LANGS+="en "
    fi
    if (echo "${files_changed}" | grep -q /zh/); then
        LANGS+="zh "
    fi
    if (echo "${files_changed}" | grep -q /pt-br/); then
        LANGS+="pt-br "
    fi
elif [[ "$#" -ne 0 ]]; then
    LANGS="$*"
else
    LANGS="en zh"
fi
```
```

Reply:
I like this. I'll add an echo message if nothing is defined to mention one can set the BRANCH if desired.